mobile/Screen/MapScreen.js

```diff
@@ -29,11 +34,47 @@ function MapScreen() {
   };
 
   return (
-    <View style={styles.container}>
-      <Text>Map Screen</Text>
-      <Button title="Test" onPress={test} />
-    </View>
+    <ScrollView style={styles.scrollView}>
+      {/* <Text>Map Screen</Text>
+      <Button title="Test" onPress={test} /> */}
+{events.map((event)=>
+      <MapEvent 
+      title={event.title}
+      date={event.date}
+      location={event.location}
+      nonprofits={event.nonprofits}
+      description={event.description}
+      navigation={navigation}
+      />
```

Comment:
This component needs to have an `id` parameter because of the warning and it prepares the component for the backend